src/cmd/cmd.cpp

```diff
@@ -155,6 +157,8 @@ void help()
     std::cout << "  --password, -p [pass]  Use [pass] as password" << std::endl;
     std::cout << "  -n                     Use netrc (5) for login" << std::endl;
     std::cout << "  --non-interactive      Do not block execution with interaction" << std::endl;
+    std::cout << "  --nonshib, -ns         Use Non Shibboleth WebDAV authentication" << std::endl;
```

Comment:
> Maybe not strictly necessary, but adding the nonshib switch (which is also theme customizable) is a lot easier and understandable from a user perspective than remembering and adding '--davpath remote.php/nonshib-webdav'.

Since we are not talking about end-user experience I do not consider this too critical.